gtk4/src/auto/string_list.rs

```diff
@@ -25,14 +22,6 @@ impl StringList {
         unsafe { from_glib_full(ffi::gtk_string_list_new(strings.to_glib_none().0)) }
     }
 
-    // rustdoc-stripper-ignore-next
-    /// Creates a new builder-pattern struct instance to construct [`StringList`] objects.
-    ///
-    /// This method returns an instance of [`StringListBuilder`](crate::builders::StringListBuilder) which can be used to create [`StringList`] objects.
-    pub fn builder() -> StringListBuilder {
```

Comment:
Instead you might want to implement the `Extend` and `FromIterator` traits here

Reply:
they are already implemented

Reply:
Oops